tools/vagrant-box-atom/upload.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/ruby
+
+require 'bundler/setup'
+require 'net/http'
+require 'vagrant_cloud'
+
+ORG = 'artefactual'
+BOX = 'atom'
+PROVIDER = 'virtualbox'
+
+path = ARGV[0]
+token = ARGV[1]
+version = ARGV[2]
+description = ARGV[3]
```

Comment:
We could use [Ruby's option parser](https://ruby-doc.org/stdlib-2.1.3/libdoc/optparse/rdoc/OptionParser.html) to improve the script argument/options management and document the script usage if needed.